### PR TITLE
fixes issue #13 - add production flag to start command

### PR DESCRIPTION
**Pull request by tdfranklin**
Ok, this should take care of it!  Let me know if you want any other changes made.

### CODE REVIEW

**Comment by brillout** [comment]
Looks good.

One thing: Currently the user can do `reframe path/to/a/dir` and the `cwd` variable will be set to `path/to/a/dir`. We could do the same for `reframe start path/to/a/dir` and `reframe start -p path/to/a/dir`. We could then get rid of the `get_cli_args` function.

**Comment by tdfranklin** [comment]
Ah thank you!  I've actually been meaning to ask you if you wanted to have that functionality or not.  do you want to always assume that commands are run in the base directory or do you want to allow the ability to pass in a path?  It will be easy enough to fix either way, but if you want to allow a full path, we'll need to change a few lines of code here and there to get expected results.

**Comment by brillout** [comment]
Thought a bit about this. Actually, I'm now thinking it to be a bad idea: It unnecessarily makes things more complicated.

I mean, there is nothing wrong with doing
~~~shell
~ cd path/to/my/app
~ reframe start
~~~

> do you want to always assume that commands are run in the base directory

So yes I think you are right. Let's simply always assume that. Thanks a lot for you feedback 👍

File: core/cli/cli.js [hunk 1]
```diff
@@ -43,10 +44,10 @@ if (typeof argValue === 'undefined') {
     process.exit(1);
 }
 
-function start() {
+function start(prod) {
     const {opts, cwd} = get_cli_args();
```
Comment (brillout) [inline core/cli/cli.js]: We can get rid of that line and of the whole `get_cli_args` function. We would then set `const cwd = process.cwd()`

**Comment by tdfranklin** [comment]
Ok, changes have been made!  Everything look alright?

**Comment by brillout** [comment]
Neat, looks good

**Pull request merged**